src/utils/common.ts

```diff
@@ -82,3 +85,29 @@ export function getOverrideIP() {
 export function testId(value: string | undefined) {
   return IS_DEVELOPMENT_BUILD || IS_TEST_MODE ? value : undefined;
 }
+
+// responsible to convert the InPlayer object to be compatible to the store
+export function processInplayerAccount(account: AccountData): Customer {
```

Comment:
These methods are probably only ever going to be used in the inplayer account service, so maybe they would be better encapsulated by being non-exported methods there?